cmd/istioctl/istioctl_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"testing"
+
+	"istio.io/manager/cmd"
+)
+
+func TestCreateInvalidFile(t *testing.T) {
+	file = "does-not-exist.yaml"
+	err := postCmd.RunE(postCmd, []string{})
```

Comment:
(style): You can limit the scope of 'err' and reduce the total lines of code by combining the 'err' assignment and check into a single line, e.g.

```
if err := postCmd.RunE(...); err != nil {
    t.Fatalf(...)
}
```